### DISCUSSION TITLE
Deployment Nodes - Nesting

### DISCUSSION THREAD

**Discussion by prasadvp** [structurizr/onpremises#113] [Q&A]
### Question

I know we can nest the deployment nodes. However, in case if i need to group the elements present across different deployment nodes, is there a way to do it? Something like this ⬇️ 

![image](https://github.com/structurizr/onpremises/assets/30220291/f03b0e5f-5a80-4af7-a437-9c84b19a1f40)

Also, by default the title of the node appear at the bottom, Is it possible to position at the top instead?

**Comment by simonbrowndotje** [comment]
This isn't supported I'm afraid.

**Comment by prasadvp** [comment]
Thank you. Do you think is it something that can be added down the line ?

**Reply by simonbrowndotje** [reply]
Unfortunately not - it's a limitation of most graph-based data structures, where a node can only have a single parent.

**Reply by prasadvp** [reply]
Thank you for clarifying.
